heat/core/tests/test_arithmetics.py

```diff
@@ -511,7 +511,7 @@ def test_prod(self):
         self.assertEqual(shape_noaxis_split_axis_neg_prod.larray.dtype, torch.float32)
         self.assertEqual(shape_noaxis_split_axis_neg_prod.split, 1)
 
-        out_noaxis = ht.zeros((1, 2, 3, 5))
+        out_noaxis = ht.zeros((1, 2, 3, 5), split=1)
```

Comment:
was this changed for completeness or another reason?

Reply:
This is because with the introduction of the sanitation module, we're a bit stricter on the output buffer, where gshape, dtype, split and device must match the result (`sanitation.sanitize_out`). I haven't been systematic in switching to the sanitation module so far, I do it when I happen to see it can be done quickly. In that case, I often have to modify the tests to define the output buffer accordingly.

Reply:
ah okay. i just wanted to make sure that the default behaviors still returned the expected results